cmd/tailscaled/tailscaled.go

```diff
@@ -464,7 +465,7 @@ func startIPNServer(ctx context.Context, logf logger.Logf, logID logid.PublicID,
 		debugMux.HandleFunc("/debug/ipn", srv.ServeHTMLStatus)
 	}
 	var lbErr syncs.AtomicValue[error]
-
+	runErrc := make(chan error, 1)
```

Comment:
Write a comment here, took me a bit to understand it. (Though I was reading on mobile, might be easier to follow on big screen)